hw7.py

```diff
@@ -0,0 +1,26 @@
+phonebook = {}
+while True:
+    user_input = input('Enter command: ')   # format command name phone
+    split_input = user_input.split()
+    command = split_input[0]
+    name = split_input[1]

```

Comment:
Your program breaks here if `list` command entered

Reply:
all commands work if inputted in format "command name phone"

Reply:
Some of the commands don't require additional arguments. For example, `add name phone` is okay because you are using all input data, but when you use `list` command you don't need more arguments. And it is not obvious for user that two more arguments should be added to `list` (like `list 1 2` or `list list list`) to make it work. To avoid this situation it is better to handle such cases and in your case you can add a simple check here:
```
command = split_input[0]
if len(split_input) > 1:
    name = split_input[1]
if len(split_input) > 2:
    phone = split_input[2]
```

Reply:
Yes, I see the problem here, but couldn't come up with a solution. Thanks for the hint! This solution is really simple and nice!